packages/docs/content/components/step-list.mdx

```diff
@@ -57,6 +57,15 @@ The following Sass variables can be overridden to customize StepList patterns:
 
 ## Guidance
 
+<ThemeContent onlyThemes={['medicare']}> 
+  
+### Design guidance 
+
+- All help drawer links will include a help icon with the link to indicate the link is a help option.
+- All help drawer links are clickable.
```

Comment:
Should we have an example that reflects this design guidance? I think the only current example of drawer + open mechanism is Drawer + Button, and it doesn't include any iconography (example just uses text).

Also, is this guidance meant to be on the StepList page or Drawer?